From 49c26ec02803a7fd5232a713ce37d75fa60e864d Mon Sep 17 00:00:00 2001
From: Dung Cao <dung@os.amperecomputing.com>
Date: Mon, 28 Apr 2025 03:01:38 +0000
Subject: [PATCH 5/6] configurations: mtjefferson: update PowerState for
 12V_FAN

The 12V_FAN should be read when host is off also.
Remove the depend on ChassisOn of PowerState for 12V_FAN configuration.

Upstream-Status: Pending

Signed-off-by: Dung Cao <dung@os.amperecomputing.com>
---
 configurations/mtjefferson_mb.json | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configurations/mtjefferson_mb.json b/configurations/mtjefferson_mb.json
index bc4ae86..968486a 100644
--- a/configurations/mtjefferson_mb.json
+++ b/configurations/mtjefferson_mb.json
@@ -248,7 +248,6 @@
                 "MinValue": 0,
                 "Name": "12V_FAN",
                 "PollRate": 15,
-                "PowerState": "ChassisOn",
                 "ScaleFactor": 0.130434783,
                 "Thresholds": [
                     {
